test/spec/roxot_analytic_spec.js

```diff
@@ -1,4 +1,3 @@
-let assert = require('assert');
 let analytic = require('../../src/adapters/analytics/roxot');
```

Comment:
For the assertions, was meaning that the test doesn't check for anything in it's current state. As it's structured could check that the `track` function was called in response to the emitted events. This can be done by setting up a sinon spy before the emits with `sinon.spy(analytic, 'track');`, then after the emits something like `sinon.assert.callCount(analytic.track, 7);`. To make this work the analytic import here would need to change to use ES2015 `import` statement, or to `let analytic = require('../../src/adapters/analytics/roxot').default;` because of how the default export is compiling. Feel free to test this according to your preferences though

Also please move this into the analytics test directory at `test/spec/adapters/analytics`, thanks